src/elements/content-uploader/ItemList.js

```diff
@@ -45,12 +51,21 @@ const ItemList = ({ items, onClick }: Props) => (
                         width={300}
                     />
                     <Column
-                        cellRenderer={actionCell}
+                        className={isResumableUploadsEnabled ? '' : 'bcu-item-list-action-column'}
+                        cellRenderer={isResumableUploadsEnabled ? resumeCell : actionCell}
```

Comment:
Can the new behaviors in `resumeCell` instead live inside `actionCell` to avoid duplication?